### PR TITLE
v1/api: move blueprints endpoints out of experimental (HMS-4200)

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Add non-experimental version of Blueprints endpoints.

Add wrappers for experimental endpoints, so we can serve both.
Wrappers will be removed once ready in #1204

Good opportunity for an API re-review :)

### CODE REVIEW

File: internal/v1/api.yaml [hunk 1]
```diff
@@ -595,7 +832,7 @@ paths:
     get:
       summary: get composes associated with a blueprint
       description: "get a collection of composes associated to a blueprint, allows for filtering by version"
-      operationId: getBlueprintComposes
+      operationId: getBlueprintComposesExperimental
```
Comment (lzap) [inline internal/v1/api.yaml:835]: Sharing the id is not supported? That would enable you to drop the wrapper functions.

**Review by lzap — APPROVED** [review]

**Review by lzap — APPROVED** [review]
Ok let’s do this, rebasing.

**Comment by ezr-ondrej** [comment]
We are :green_apple: :)

**Pull request merged**